mkdocs_literate_nav/plugin.py

```diff
@@ -53,60 +55,78 @@ def on_files(self, files: mkdocs.structure.files.Files, config: mkdocs.config.Co
                 tab_length=self.config["tab_length"],
             ),
         )
-        self._files = files
+
+        if self.config["explicit"]:
+
+            def is_file_in_nav_file(f) -> bool:
+                if (
+                    str(f.src_uri) in self.nav_parser.files_in_nav
+                    or f not in files.documentation_pages()
+                ):
+                    return True
+                log.warning(f"File excluded from navigation file: {f.src_uri}")
```

Comment:
@flynneva Can you have this not emit a warning? This would fail strict doc builds.